docs/MAINTAINERS.adoc

```diff
@@ -0,0 +1,230 @@
+* Proper changelog labels are set so that the changelog can be generated automatically.
+ List of labels we use for changelog generation is available link:https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.yml[here].
+* If the change needs administrators to be aware of it when upgrading, the `upgrade-guide-needed` label is present
+  and there is a `Proposed upgrade guidelines` section in the PR title
+  (link:https://github.com/jenkinsci/jenkins/pull/4387[example]).
+  This is usually the case when a data migration occurs, a feature has been removed, a significant behavior change is introduced (including when there is a way to opt out),
+  or in general when we expect at least a large minority of admins to benefit from knowing about the change, e.g. to apply a new option.
+* It would make sense to backport the change to LTS, a Jira issue should exist and be labeled as `lts-candidate`
+
+==== Step 2: Is it a good time to merge?
+
+link:[https://jenkins.io/security/](Jenkins security updates) are coordinated with the LTS calendar, and if the weekly introduces regressions, users of that line may have to choose between security fixes, and a working Jenkins.
+The Jenkins security team will usually send a "pre-announcement" to link:https://groups.google.com/forum/#!forum/jenkinsci-advisories[the advisories list] on Wednesday or Thursday the week before release, but that's not always doable.
+For these reasons, the following changes should not be merged during the week before LTS releases (weeks 3, 7, 11, 15, etc. on the page linked above):
+
+* Changes that could be considered risky (relatively high risk of introducing regressions), as they could make users of Jenkins weekly releases choose between getting security fixes, and having a functioning Jenkins
+* Very large changes (in terms of lines changed), because the Jenkins security team needs to prepare security fixes for the weekly release line in a very short period of time.
+
+If the change is ready but it is not a good time, consider labeling the pull request with the `on-hold` label.
+If a merge is needed, please consult with the Jenkins Security officer
+
+==== Step 3: Marking for merge
+
+Once the checklist is passed, a Core PR Reviewer or a Maintainer can mark the pull request for merge.
+
+* `ready-for-merge` label is set
+* An explicit comment is added to the pull request so that other repository watchers are notified.
+  Example: _Thanks to all contributors! We consider this change as ready to be merged towards the next weekly release. It may be merged after 24hours if there is no negative feedback_
+
+==== Step 4: Merge!
+
+A Core Maintainer merges the change after allowing sufficient time for comment (if needed).
+After that the change will be landed in the next weekly release.
+LTS Backporting, if needed, will be handled separately by the release team.
+
+Keep in mind that we set 
```

Comment:
Either an incomplete sentence or the start of the next section heading `Exceptions`.  Assumed an incomplete sentence.
```suggestion
```